ui/app/AppLayouts/Profile/stores/DevicesStore.qml

```diff
@@ -46,6 +46,6 @@ QtObject {
     }
 
     function inputConnectionStringForBootstrapping(connectionString) {
-        return root.devicesModule.inputConnectionStringForBootstrapping(connectionString)
+        root.devicesModule.inputConnectionStringForBootstrapping(connectionString)
```

Comment:
Is it ok that it doesn't return the value anymore? Not sure if it was ever used, but it affects current behaviour.

Reply:
iirc, we never used it in the QML at all, that's why I felt comfortable to remove it